dockstore-integration-testing/src/test/java/io/dockstore/client/cli/Ga4GhTRSAPIWorkflowIT.java

```diff
@@ -132,6 +140,23 @@ public void testGa4ghEndpointForComplexWdlWorkflow() {
             // TRS paths are relative
             assertTrue(sourceFiles.stream().anyMatch(sf -> sf.getAbsolutePath().equals("/" + path)));
         });
+
+        // test zip download via GA4GH TRS 2.0.1
+        Ga4Ghv20Api ga4Ghv20Api = new Ga4Ghv20Api(getOpenAPIWebClient(USER_2_USERNAME, testingPostgres));
+        final List<io.dockstore.openapi.client.model.ToolFile> toolFiles = ga4Ghv20Api.toolsIdVersionsVersionIdTypeFilesGet("#workflow/" + refresh.getFullWorkflowPath(),
+            DescriptorTypeWithPlain.WDL.toString(), GATK_SV_TAG, null);
+        byte[] arbitraryURL = CommonTestUtilities.getArbitraryURL(
+            "/ga4gh/trs/v2/tools/" + URLEncoder.encode("#workflow/" + refresh.getFullWorkflowPath(), StandardCharsets.UTF_8) + "/versions/" + URLEncoder.encode(GATK_SV_TAG, StandardCharsets.UTF_8)
+                + "/" + DescriptorTypeWithPlain.WDL
+                + "/files?format=zip", new GenericType<>() {
+                }, ownerWebClient);
+        File tempZip = File.createTempFile("temp", "zip");
```

Comment:
It's just tests, but you could still do `tempZip.deleteOnExit()`, or explicitly delete below.